crates/holochain/src/conductor/conductor/builder.rs

```diff
@@ -170,14 +170,24 @@ impl ConductorBuilder {
             Some(keystore.lair_client()),
         );
 
-        let (holochain_p2p, p2p_evt) =
-            match holochain_p2p::spawn_holochain_p2p(network_config, tls_config, host).await {
-                Ok(r) => r,
-                Err(err) => {
-                    tracing::error!(?err, "Error spawning networking");
-                    return Err(err.into());
-                }
-            };
+        let network_compat = todo!(
+            "get network compat params from conductor state, but we have to write a way to do that before initializing the conductor..."
+        );
```

Comment:
not sure conductor state is where we'll get this... i guess if dpki is hard-coded for now, the dna hash will just be hard-coded? then later, we'll have to get it from the config?

Reply:
In the DPKI branch, conductor state stores the installed conductor services (they are akin to installed apps) so they can be reinitialized again on restart

Reply:
:upside_down_face: